calyx/src/passes/compile_sync_without_sync_reg.rs

```diff
@@ -0,0 +1,186 @@
+use crate::errors::{CalyxResult, Error};
+use crate::ir::traversal::{Action, Named, VisResult, Visitor};
+use crate::ir::RRC;
+use crate::ir::{self, GetAttributes};
+use crate::{build_assignments, guard, structure};
+use std::collections::HashMap;
+
+#[derive(Default)]
+pub struct CompileSyncWithoutSyncReg;
+
+impl Named for CompileSyncWithoutSyncReg {
+    fn name() -> &'static str {
+        "compile-sync-without-sync-reg"
```

Comment:
This name is a little cumbersome. Since this is a default pass, can we call it `compile-sync` and other one as `compile-sync-with-sync-reg`?